packages/devtools_app/lib/src/debugger/scripts.dart

```diff
@@ -148,10 +150,36 @@ class ScriptPickerState extends State<ScriptPicker> {
               ),
               const SizedBox(width: densePadding),
               Expanded(
-                child: Text(
-                  node.name,
-                  maxLines: 1,
-                  overflow: TextOverflow.ellipsis,
+                child: Row(
+                  children: [
+                    Expanded(
```

Comment:
we can probably remove this Row and Expanded since Expanded is the only child of Row